spec/javascripts/health-care/components/personal-information/VeteranAddressSection.spec.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import SkinDeep from 'skin-deep';
+
+import VeteranAddressSection from '../../../../../_health-care/_js/components/personal-information/VeteranAddressSection';
+
+describe('<VeteranAddressSection>', () => {
+  const mockAddress = [{
+    street: null,
+    city: null,
+    country: null,
+    state: null,
+    zipcode: null,
+  }];
+  it('Sanity check the component renders', () => {
+    const tree = SkinDeep.shallowRender(<VeteranAddressSection data={{ address: mockAddress }}/>);
+    const vdom = tree.getRenderOutput();
+    expect(vdom).to.have.property('type', 'fieldset');
+  });
+
+  it('does not include `error` prop when Confirm Email matches Email', () => {
```

Comment:
Surround these two by a `describe('Email confimration', () => { ... }`.  Then you can shorten the `it` description. It also produces slightly nicer output on the test log.